GrandCentralBoard/Core/Configuration/ConfigurationRefresher.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Created by Oktawian Chojnacki on 21.04.2016.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import Foundation
+
+
+public final class ConfigurationRefresher: Schedulable {
+
+    private let scheduler: SchedulingJobs
+    private let fetcher: ConfigurationFetching
+    private weak var configuree: Configurable?
+
+    public let interval: NSTimeInterval
+    public let selector: Selector = #selector(fetch)
```

Comment:
Seriously selector? Why didn't you wrap `NSTimer` calls in `Scheduler` in blocks?